docs/source/userguide/creating_a_new_design.rst

```diff
@@ -0,0 +1,163 @@
+    #. When you reach your desired location for the Block within the Layout Panel release the mouse button.
+
+After a brief pause the Block Palette icon is replaced by a full representation of the selected Block, showing:
+
+    * The formal Block identifier.
+    * An optional, configurable descriptive label (initially containing default text).
+    * `Source ports <source_port_>` available to the Block, including their type.
+    * `Sink ports <sink_port_>` available to the Block, including their type.
+
+After adding a Block to the Layout Panel it can be selected by hovering over it and clicking the left mouse button.  Upon selection the Block Information panel presenting each `attribute_` and `method_` available to that Block is displayed in the right-hand panel of the web interface.
+
+
+.. _removing_a_block_from_a_design_:
+
+Removing a Block from a Design
+---------------------------------
+
+If a `block_` has been added to a `design_` erroneously, or is no longer required within the current Design it can be removed as follows:
+
+    #. Select the Block to be removed by hovering over it and clicking the left mouse button.  Upon selection a 'garbage can' icon is displayed at the bottom of the Layout Panel.
+    #. While holding down the left mouse button drag the Block over the garbage can icon.  The icon is highlighted.
+    #. Release the left mouse button.
+
+
+Working with the Block Palette
+------------------------------
+
+The Block Palette contains a list of each `block_` available to a `design_` based on pre-defined constraints imposed by the underlying hardware infrastructure associated with the system.
+
+When a Block is selected from the Block Palette for `inclusion <adding_a_block_to_a_design_>` in a Design it is removed from the Block Palette to ensure it is not included more than once.  If all Blocks of a particular type have been added to a Design it is not possible to add any more as the underlying hardware implementation will not be able to represent them.
+
+If a Block is `removed <removing_a_block_from_a_design_>` from a Design it is immediately available again for selection in the Block Palette.
+
+
+Specifying Block Attributes
+---------------------------
+
+The behaviour of a `block_` is defined via its `attributes <attribute_>`.  Attributes are pre-defined based on the function of the Block and may include default values providing a starting point for later implementation-time customisation.  A full list of the attributes associated with each Block available from the Block Palette can be found in the `PandABlocks-FPGA`_ documentation.
+
+Types of Attributes
+^^^^^^^^^^^^^^^^^^^
+
+`block_` attributes come in two forms:
+
+    #. An `immutable_attribute_` is one that once set within the `Design` cannot be modified by any action within the system.  It holds a constant value until manually modified as part of a controlled Design change.
```

Comment:
Need to discuss

Reply:
I've removed 'immutable ' and 'writeable' attribute descriptions and replaced with 'Parameter', 'Input', 'Output' and 'Readback', each of which describes the read/write properties of the attribute type individually